app/javascript/app/pages/country-compare/country-compare-styles.scss

```diff
@@ -12,3 +9,15 @@
     border-right: 1px solid $theme-border;
   }
 }
+
+.compareWrapper {
```

Comment:
This starting to be a repetitive task... I had to do the same on all of the other pages so maybe worth it to promote to app level so we don't have to do it over and over again. What do you think?

Reply:
It makes sense.  If you agree I'll address it on the ndc's branch I'm working at this moment.